network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/research_detail_card.html

```diff
@@ -24,8 +24,8 @@
       {% if research_detail_page.get_research_author_names and not hide_author_names %}
         <div class="tw-px-2 tw-min-w-0 tw-truncate" >{{ research_detail_page.get_research_author_names|join:', ' }}</div>
       {% endif %}
-      {% if research_detail_page.get_related_topic_names %}
-        <div class="tw-pl-2 tw-min-w-0 tw-truncate {% if hide_topics_on_mobile %} tw-hidden small:tw-block {% endif %}">{{ research_detail_page.get_related_topic_names|join:' / ' }}</div>
+      {% if research_detail_page.get_related_topic_names and not hide_related_topics %}
+        <div class="tw-pl-2 tw-min-w-0 tw-truncate {% if hide_related_topics_on_mobile %} tw-hidden small:tw-block {% endif %}">{{ research_detail_page.get_related_topic_names|join:' / ' }}</div>
```

Comment:
Per request of https://github.com/mozilla/foundation.mozilla.org/pull/8636#discussion_r869609955

Reply:
**Issue** We will also need to rename `hide_topics_on_mobile` to `hide_related_topics_on_mobile` in the library page template.